src/js/tech/flash.js

```diff
@@ -69,6 +70,15 @@ class Flash extends Tech {
   createEl() {
     let options = this.options_;
 
+    // If video.js is hosted locally you should also set the location
+    // for the hosted swf, which should be relative to the page (not video.js)
+    // Otherwise this adds a CDN url.
+    // The CDN also auto-adds a swf URL for that specific version.
+    if (!options.swf) {
+      let protocol = ('https:' === window.location.protocol ? 'https://' : 'http://');
+      options.swf = protocol+'vjs.zencdn.net/swf/'+pkg.dependencies['videojs-swf']+'/video-js.swf';
```

Comment:
will the version string of the swf always be just the version? Could we at some point switch to using `^` or anything that could cause this to break?